src/tools/genpolicy/src/policy.rs

```diff
@@ -425,6 +425,9 @@ pub struct CommonData {
 pub struct ClusterConfig {
     /// Pause container image reference.
     pub pause_container_image: String,
+    /// Whether or not the cluster uses the guest pull mechanism
+    /// See issue https://github.com/kata-containers/kata-containers/issues/11162
```

Comment:
Would recommend one line generally explaining the situation here (in guest pull, host can't look into layers to determine GID)